30/08/2019 11:16:14 AM Python Script is: morePVs.py
30/08/2019 11:16:14 AM study_name = test_capex2
30/08/2019 11:16:14 AM Running Scenario number 1 
30/08/2019 11:16:14 AM ***************Exception!!! PV /Users/mikeroberts/Documents/python/morePVs/DATA_EN_5/pv_profiles/test_2day_pv.csv index does not align with load 
30/08/2019 11:16:14 AM 


 Exception !!!!!!
Traceback (most recent call last):
  File "/Users/mikeroberts/Documents/python/morePVs/morePVs.py", line 2654, in main
    runScenario(s)
  File "/Users/mikeroberts/Documents/python/morePVs/morePVs.py", line 2574, in runScenario
    scenario = Scenario(scenario_name=scenario_name)
  File "/Users/mikeroberts/Documents/python/morePVs/morePVs.py", line 1863, in __init__
    sys.exit("PV has bad timeseries")
SystemExit: PV has bad timeseries
